src/utils/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+const articlesApi = axios.create({
+  baseURL: 'https://nc-backend-project-nc-news.onrender.com/api'
+});
+
+export const getCategories = () => {
```

Comment:
Let's also remove this (regarding topics and categories for now - again not relevant to this ticket)